### PR TITLE
Module 3 Block 4: React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by Elijahdre**
## In this PR, I have implemented the following tasks:

- [ ] Bootstrap app UI.
- [ ] Fetch rocket data using async thunk.
- [ ] Render rocket list on rocket.js.
- [ ] Display missions list.
- [ ] Implement mission joining.
- [ ] Implement testing for components.

### CODE REVIEW

File: src/components/profile.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <div className="lg:w-1/2 flex flex-col gap-4">
+        <table>
+          <thead>
+            <tr>
+              <td className="text-4xl">My Rockets</td>
+            </tr>
```
Comment (aceDavon) [inline src/components/profile.js]: Here on your missions page, you have a label "My Rockets" on the missions table, You should change this to "My mission" just as required by the curriculum.

**Review by aceDavon — CHANGES_REQUESTED** [review]
## Changes required :recycle: 
### Hi @Elijahdre 
almost there 2

You have done a great job so far, However, there are a few changes you should implement to proceed with this project, Please check the comments on the files below:

## To Highlight :green_circle: 

- Great Interface Implementation,
- Linters are passing,
- Correct use of git workflow
- Professional Project documentation

_Be sure to checkout for comments marked as [optional] and implement those at your disposal._

**Happy Coding :tada:**

File: src/components/profile.js (hunk 1 above)
Comment (Elijahdre) [reply src/components/profile.js]: Thanks for bringing my attention to this error 😄 .

**Review by aceDavon — APPROVED** [review]
## Project Approved :tada: 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f7352396e696c457a6e446f536c67613636372f67697068792e676966

There is nothing left to say, Your Pull request is completely good to be merged.

## Highlight(s)

- Updated LinkedIn link
- Clean Project documentation

**Happy coding :100: **

**Pull request merged**